frigate/comms/mqtt.py

```diff
@@ -89,6 +89,18 @@ def _set_initial_topics(self) -> None:
                 "OFF",
                 retain=False,
             )
+            self.publish(
+                f"{camera_name}/birdseye/state",
+                "ON" if camera.birdseye.enabled else "OFF",
+                retain=True,
+            )
+            self.publish(
+                f"{camera_name}/birdseye_mode/state",
```

Comment:
if we are going to use this here then the other topic should be `birdseye_mode/set`

Reply:
To be clear ptz was treated differently because there's no way to guarantee its current state because the camera can be moved without frigate knowing